src/nus/cs2103/addressbook/AddressBook.java

```diff
@@ -475,32 +552,38 @@ private static String getUserInput() {
 
     /**
      * Shows a message to the user
-     *
-     * @param message to show
      */
     private static void showToUser(String message) {
         System.out.println(message);
     }
 
     /**
-     * Formats and shows an indexed list of persons to the user.
+     * Shows the list of persons to the user.
+     * The list will be indexed, starting from 1.
      *
-     * @param persons to show
      */
     private static void showToUser(List<String[]> persons) {
+        String listAsString = getDisplayString(persons);
+        showToUser(listAsString);
+        updateLatestViewedPersonListing(persons);
+    }
+
+    /**
+     * Returns the display string representation of the list of persons.
+     */
+    private static String getDisplayString(List<String[]> persons) {
```

Comment:
Sure. The word 'display' implies it is 'one-indexed' (because it is known that display is one-indexed)?